src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryOptions.java

```diff
@@ -97,6 +97,18 @@ public class RepositoryOptions extends OptionsBase {
               + " still run an arbitrary executable that accesses the Internet.")
   public boolean disableDownload;
 
+  @Option(
+      name = "enable_native_repo_rules",
```

Comment:
hmm, I actually went back and forth on this, since we have flags like `--enable_bzlmod` and `--enable_workspace`... but I went with `--incompatible_enable_native_repo_rules` anyway. not sure what the convention should be here.

Reply:
I prefer `--incompatible_disable_native_repo_rules`, I think the convention is that what follows `incompatible_` is the actual breaking behaviour, here the breaking change is "disable native repo rules" instead of "enable native repo rules".

Check the other ones: https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+incompatible+is%3Aclosed+label%3Aincompatible-change

Reply:
SGTM -- done

Reply:
Thanks!